src/systems/systems/src/hydraulic/linear_actuator.rs

```diff
@@ -70,51 +84,87 @@ struct CoreHydraulicForce {
 
     last_control_force: Force,
 
-    force: Force,
+    force_raw: Force,
+    force_filtered: LowPassFilter<Force>,
+    max_force: Force,
+
+    pid_controller: PidController,
 }
 impl CoreHydraulicForce {
-    const DEFAULT_I_GAIN: f64 = 0.2;
-    const DEFAULT_P_GAIN: f64 = 0.05;
-    const DEFAULT_FORCE_GAIN: f64 = 200000.;
+    const OPEN_LOOP_GAIN: f64 = 1.;
+
+    const MIN_PRESSURE_TO_EXIT_POSITION_CONTROL_PSI: f64 = 500.;
+    const MIN_PRESSURE_TO_ALLOW_POSITION_CONTROL_PSI: f64 = 700.;
 
     fn new(
+        _: &mut InitContext,
```

Comment:
This is not necessary. You can add the `InitContext` back in in the future when you require it.

Reply:
Added this for tuning purpose only, and thought I would get mad if I had to remove it to again cascade it again if we add new init stuff....   Will remove but if I have to add it again I'll find you!!!! :D